.github/workflows/lintfix.yml

```diff
@@ -0,0 +1,91 @@
+name: lintfix
+
+on:
+  # enable normal pr for testing, needs to be disabled!!!
+  pull_request:
+  pull_request_target:
+
+env:
+  YARN_MODULES_CACHE_KEY: v1
+  YARN_PACKAGE_CACHE_KEY: v1
+  YARN_CACHE_FOLDER: .cache/yarn
+  NODE_VERSION: 12
+  # https://github.com/chalk/supports-color/issues/106
+  FORCE_COLOR: true
+  FIX_AUTHOR_EMAIL: renovate@whitesourcesoftware.com
+
+jobs:
+  lint-fix:
+    runs-on: ubuntu-latest
+
+    # lint shouldn't need more than 10 min
+    timeout-minutes: 15
+
+    steps:
+      - name: Set up Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v1.4.4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Init platform
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.symlinks true
+          git config --global user.email $FIX_AUTHOR_EMAIL
+          git config --global user.name 'Renovate Bot'
+          npm config set scripts-prepend-node-path true
+          echo "Node $(node --version)"
+          echo "Yarn $(yarn --version)"
+
+      - name: checkout base
+        uses: actions/checkout@v2.3.3
+        with:
+          persist-credentials: false # do not persist credentials
+
+      - name: Cache Yarn packages
+        id: yarn_cache_packages
+        uses: actions/cache@v2.1.2
+        with:
+          path: ${{ env.YARN_CACHE_FOLDER }}
+          key: ${{ env.YARN_PACKAGE_CACHE_KEY }}-${{ runner.os }}-yarn_cache-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ env.YARN_PACKAGE_CACHE_KEY }}-${{ runner.os }}-yarn_cache-
+
+      - name: Installing dependencies
+        run: yarn install --frozen-lockfile
+
+      # - name: fetch pr head
+      #   run: git fetch
+
+      - name: checkout pr head
+        uses: actions/checkout@v2.3.3
+        with:
+          persist-credentials: false # do not persist credentials
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          clean: false
+
+      - name: Lint
+        run: |
+          docker run --rm -t -u "$UID:0" \
+           -v "${PWD}:/code" \
+           -v "$(mktemp -d -t ci-XXXXXXXXXX):/code/.git:ro" \
+           -v "$(mktemp -d -t ci-XXXXXXXXXX):/tmp/${UID}" \
+           -w /code \
+           -e HOME=/tmp/${UID} \
+           "renovate/node:${{ env.NODE_VERSION}}" \
+           npx yarn lint-fix
```

Comment:
GitHub Actions has [workflow commands](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions), which can manipulate the environment for subsequent actions. The Docker output should be redirected to `null` or a file, which can be uploaded via artifact.

Reply:
the `::set-env` command was deprecated some time ago, is it still working? 🤔 also `$GITHUB_ENV` file is not available to docker container.

but sure, for safety we can redirect docker output to file and upload as asset to review if required

Reply:
AFAIK, they still work but only produce warnings: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/.

Reply:
ok, added redirect to file